app/views/nav.html

```diff
@@ -1,17 +1,64 @@
-<nav class="navbar navbar-default navbar-fixed-top" role="navigation">
+<nav
+    class="navbar navbar-default navbar-fixed-top"
+    role="navigation">
   <div class="container-fluid">
     <div class="navbar-header">
-      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#navbar-collapse">
-        <span class="sr-only">Toggle navigation</span>
-        <span class="icon-bar"></span>
-        <span class="icon-bar"></span>
-        <span class="icon-bar"></span>
+      <button
+          type="button"
+          class="navbar-toggle"
+          data-toggle="collapse"
+          data-target="#navbar-collapse">
+        <i class="fa fa-bars"></i>
       </button>
-      <a id="logo" class="navbar-brand" href="/cms/app/list/"><img ng-src="{{ NAV_LOGO }}"></a>
+      <a
+          id="logo"
+          class="navbar-brand"
+          href="/cms/app/list/">
+        <img ng-src="{{ NAV_LOGO }}">
+      </a>
     </div>
 
-    <div class="collapse navbar-collapse" id="navbar-collapse">
+    <div
+        class="collapse navbar-collapse"
+        id="navbar-collapse">
+
       <ul class="nav navbar-nav">
+
+        <li
+          class="collapse navbar-collapse"
+          id="navbar-collapse">
+
+        <div class="dropdown">
```

Comment:
Mostly spacing except for the following `Create` dropdown, which nests a bunch of navigation items.